src/deadline/client/api/__init__.py

```diff
@@ -4,6 +4,7 @@
     "login",
     "logout",
     "create_job_from_job_bundle",
+    "_hash_attachments",
```

Comment:
This `api` namespace is generally about calling the web APIs. Hashing job attachments files seems like a more local operation, maybe there's another place it would fit better?

Reply:
It was originally in the CLI `_submit_job_bundle.py`

One of my ideas was to completely move it to job attachments sub folder. but the current definition does not lend itself easily to be there. I can also move it to a `utils` like location?

Reply:
Yes, it was there as an implementation detail. What's the purpose to move it? Could it stay an internal detail the way it is now, or does it need to be a function that the library exposes for external use?

Reply:
Good point - I'm not sure why we added it to `__all__`, now that I think about it, its `_` means it is internal. I'll remove this from `__all__` altogether.

Its one of the things I picked up from the original implementation branch while cutting it up for merging. Great catch!

I will still extract this method from the bundle submission file to a separate file. There will be a few places that leverage this shared code.